src/code/utilities/translate.js

```diff
@@ -0,0 +1,29 @@
+import translations from './lang';
+
+const defaultLang = "en_us",
+      varRegExp = /\$\{\s*(\d+)\s*\}/g,
+      error = "** TRANSLATION ERROR **";
+
+const translateString = (key, lang) => translations[lang][key] || key;
```

Comment:
Perhaps a test for existence of translations[lang] in case of request for language for which there are no translations.